examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal

```diff
@@ -11,88 +11,67 @@ type Student record {
-    io:println("Call stored procedure `GetCount`.");
-    io:println("Age of the student with id '1' : ", id.get(int));
-    io:println("Total student count: ", totalCount.get(int));
-    check retCall2.close();
-
-    // Invokes the stored procedure, which returns the data.
-    sql:ProcedureCallResult retCall3 = 
-            check mysqlClient->call(`{CALL GetStudents()}`, [Student]);
-    io:println("Call stored procedure `GetStudents`.");
-
-    // Processes the returned result stream.
-    stream<record {}, sql:Error?>? result = retCall3.queryResult;
-    if result is stream<record {}, sql:Error?> {
-        stream<Student, sql:Error?> studentStream =
-                <stream<Student, sql:Error?>>result;
-        check from Student student in studentStream
-            do {
-                io:println("Student details: ", student);
-            };
-    }
-    check retCall3.close();
+
+    // The stored procedure is invoked.
+    sql:ProcedureCallResult result =
+        check mysqlClient->call(`{CALL GetCount(${id}, ${totalCount})}`);
+
+    // Process procedure call parameters.
+    int studentId = check id.get();
+    int total = check totalCount.get();
+
+    io:println(`Age of the student with id '1' : ${studentId}`);
+    io:println(`Total student count: ${total}`);
+
+    // Closes the procedure call result to release resources.
+    check result.close();
+
+    // Closes the MySQL client.
+    check mysqlClient.close();
 
     // Performs the cleanup after the example.
-    check afterExample(mysqlClient);
+    check cleanup();
 }
 
 // Initializes the database as a prerequisite to the example.
-function beforeExample() returns sql:Error? {
+function initialize() returns sql:Error? {
     mysql:Client mysqlClient = check new (user = "root", password = "Test@123");
 
     // Creates a database.
-    _ = check mysqlClient->execute(`CREATE DATABASE MYSQL_BBE`);
+    _ = check mysqlClient->execute(`CREATE DATABASE STUDENT`);
 
     // Creates a table in the database.
-    _ = check mysqlClient->execute(`CREATE TABLE MYSQL_BBE.Student
+    _ = check mysqlClient->execute(`CREATE TABLE STUDENT.Student
             (id INT AUTO_INCREMENT, age INT, name VARCHAR(255),
             PRIMARY KEY (id))`);
 
     // Creates the necessary stored procedures using the execute command.
-    _ = check mysqlClient->execute(`CREATE PROCEDURE
-        MYSQL_BBE.InsertStudent (IN pName VARCHAR(255), IN pAge INT)
-        BEGIN INSERT INTO Student(age, name) VALUES (pAge, pName); END`);
-    _ = check mysqlClient->execute(`CREATE PROCEDURE MYSQL_BBE.GetCount
+    _ = check mysqlClient->execute(`CREATE PROCEDURE STUDENT.GetCount
         (INOUT pID INT, OUT totalCount INT) BEGIN SELECT age INTO pID FROM
         Student WHERE id = pID; SELECT COUNT(*) INTO totalCount FROM Student;
         END`);
-    _ = check mysqlClient->execute(`CREATE PROCEDURE
-        MYSQL_BBE.GetStudents() BEGIN SELECT * FROM Student; END`);
+    // Insert necessary data
```

Comment:
```suggestion
    // Inserts necessary data.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)